mypyc/build.py

```diff
@@ -546,10 +546,8 @@ def mypycify(
             "-Wno-unused-variable",
             "-Wno-unused-command-line-argument",
             "-Wno-unknown-warning-option",
+            "-Wno-unused-but-set-variable",
```

Comment:
Just to confirm, do compilers generally ignore unused -W flags?

Reply:
The config `-Wno-unknown-warning-option` on the line before should silence any errors about this I believe.

Reply:
Apparently I spoke too soon :/

```
cc1: error: unrecognized command line option ‘-Wno-unknown-warning-option’
cc1: error: unrecognized command line option ‘-Wno-unused-command-line-argument’
```